bin/tests/it/k8s.rs

```diff
@@ -870,13 +870,18 @@ async fn create_agent_startup_lease_list(client: Client, name: &str, namespace:
         .unwrap();
 }
 
-async fn create_agent_feature_lease(client: Client, namespace: &str) {
+async fn create_agent_feature_lease(
+    client: Client,
+    namespace: &str,
+    lease_name: &str,
```

Comment:
Nit: maybe be a little more specific in the naming here? I'm just thinking about the confusion between the startup leases and these leases.

Reply:
as in the function name or the parameter lease_name?